packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -399,6 +505,19 @@ function inputToBoolean(value: unknown): boolean {
   return value != null && `${value}` !== 'false';
 }
 
+/**
+ * Sorts provided config breakpoints and uses defaults.
+ */
+function processConfig(config: ImageConfig): ImageConfig {
+  let sortedBreakpoints: {subViewportBreakpoints?: number[], viewportBreakpoints?: number[]} = {};
+  if (config.subViewportBreakpoints) {
```

Comment:
nit: let's do object destructing here to same some bytes (after minificattion all property names remain the same, when const names are minified):
```suggestion
  const {viewportBreakpoints, subViewportBreakpoints} = config;
  let sortedBreakpoints: {subViewportBreakpoints?: number[], viewportBreakpoints?: number[]} = {};
  if (subViewportBreakpoints) {
```